smartfile/__init__.py

```diff
@@ -1,16 +1,17 @@
-from netrc import netrc
 import os
 import re
+import shutil
 import time
 import urllib
+
+from netrc import netrc
```

Comment:
It appears that we allow the use of an NETRC file to be passed to the basic client. 

Perhaps a new ticket/issue to remove this would finally be appropriate?

https://github.com/smartfile/client-python/blame/04ff2ce406d98f3a299ab956c476ba27feae8956/smartfile/__init__.py#L135